doc/src/tutorials/biomechanics/biomechanics.rst

```diff
@@ -0,0 +1,1376 @@
+activation. In our case, we will use a first order ordinary differential
+equation that gives a smooth, but delayed activation :math:`a(t)` from the
+excitation :math:`e(t)`.
+
+.. plot::
+   :format: doctest
+   :include-source: True
+   :context: close-figs
+   :nofigs:
+
+   >>> from sympy.physics._biomechanics import FirstOrderActivationDeGroote2016
+   >>> muscle_activation = FirstOrderActivationDeGroote2016.with_defaults('muscle')
+
+The activation model has a state variable :math:`\mathbf{x}`, input variable
+:math:`\mathbf{r}`, and some constant parameters :math:`\mathbf{p}`:
+
+.. plot::
+   :format: doctest
+   :include-source: True
+   :context: close-figs
+   :nofigs:
+
+   >>> muscle_activation.x
+   Matrix([[a_muscle(t)]])
+   >>> muscle_activation.r
+   Matrix([[e_muscle(t)]])
+   >>> muscle_activation.p
+   Matrix([
+   [0.015],
+   [ 0.06],
+   [   10]])
+
+These are associated with its first order differential equation :math:`\dot{a}
+= f(a, e, t)`:
+
+.. plot::
+   :format: doctest
+   :include-source: True
+   :context: close-figs
+   :nofigs:
+
+   >>> muscle_activation.rhs()
+   Matrix([[((1/2 - tanh(10*a_muscle(t) - 10*e_muscle(t))/2)/(0.0225*a_muscle(t) + 0.0075) + 16.6666666666667*(3*a_muscle(t)/2 + 1/2)*(tanh(10*a_muscle(t) - 10*e_muscle(t))/2 + 1/2))*(-a_muscle(t) + e_muscle(t))]])
+
+With the pathway and activation dynamics, the musculotendon model created using
+them both and needs some parameters to define the muscle and tendon specific
+properties. You need to specify the tendon slack length, peak isometric force,
+optimal fiber length, maximal fiber velocity, optimal pennation angle, and
+fiber damping coefficients.
+
+.. TODO : How do we know this is a rigid tendon model?
+
+.. plot::
+   :format: doctest
+   :include-source: True
+   :context: close-figs
+   :nofigs:
+
+   >>> from sympy.physics._biomechanics import MusculotendonDeGroote2016
+
+   >>> F_M_max, l_M_opt, l_T_slack = sm.symbols('F_M_max, l_M_opt, l_T_slack', real=True)
+   >>> v_M_max, alpha_opt, beta = sm.symbols('v_M_max, alpha_opt, beta', real=True)
+
+   >>> muscle = MusculotendonDeGroote2016(
+   ...     'muscle',
+   ...     muscle_pathway,
+   ...     muscle_activation,
+   ...     tendon_slack_length=l_T_slack,
+   ...     peak_isometric_force=F_M_max,
+   ...     optimal_fiber_length=l_M_opt,
+   ...     maximal_fiber_velocity=v_M_max,
+   ...     optimal_pennation_angle=alpha_opt,
+   ...     fiber_damping_coefficient=beta,
+   ... )
+   ...
+
+.. TODO : Explain why the rhs() is different for the muscle than the activation.
+.. TODO : Needs explanation about rigid tendon
+
+Because this musculotendon actuator has a rigid tendon model, it has the same
+state and ordinary differential equation as the activation model:
+
+.. plot::
+   :format: doctest
+   :include-source: True
+   :context: close-figs
+   :nofigs:
+
+   >>> muscle.musculotendon_dynamics
+   MusculotendonFormulation.RIGID_TENDON
```

Comment:
This is passing for me locally with both `pytest --doctest-only` and `bin/doctest`. In CI it's failing saying that the got value `0`. `MusculotendonFormulation` is an `Enum` and the `RIGID_TENDON` member does have value `0`.

Reply:
I've never used an enum in python, so I'm not sure what is supposed to occur.

Reply:
There was a change between Python 3.10 and 3.11 in how [`__str__` is implemented for `StrEnum` class](https://github.com/python/cpython/issues/84247). To ensure that this doctest passes on all Python version we'll need to override the default `__str__`.

Reply:
Addressed in https://github.com/sympy/sympy/pull/25525/commits/2e59f6ad687155c43de9401a9823c00650450bb2 and https://github.com/sympy/sympy/pull/25525/commits/8bb46c5a66332166fbf1dc35efb7069ba497c54d.